scripts/planning/sprint-planning.sh

```diff
@@ -52,7 +52,7 @@ function github_milestone()
 
 function jira_milestone()
 {
-  milestone_data=`curl -s -X GET -H "Content-Type: application/json" "https://issues.jboss.org/rest/api/2/search?jql=project=CDK%20AND%20component=minishift%20AND%20fixVersion=${milestone}"`
```

Comment:
So we just now removed the minishift from the component and only care about the CDK and the given milestone :+1:

Reply:
@praveenkumar yes because we few components other than `minishift`.